src/ast/helpers.rs

```diff
@@ -622,6 +622,19 @@ pub fn preceded_by_continuation(stmt: &Stmt, locator: &Locator) -> bool {
     false
 }
 
+// Return the `Range` of the first `Tok::Colon` token in a `Range`.
```

Comment:
Doc comments should start with three slashes (`///`) and `Tok::Colon` can be wrapped in [] so that rustdoc turns it into a link (you can check that with `cargo doc --no-deps --open`).

Reply:
Thanks for the catch!

Reply:
So if we wrap `Tok::Colon` and `Range` in `[]`, does rustdoc creates links to RustPython's doc?